doc/README.md

```diff
@@ -0,0 +1,95 @@
+    1. that you have a [CADC account](http://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/en/auth/request.html), which is configured by CADC to have read and write access to a CAOM `COLLECTION`.
+
+1. This description uses the parameters `TEST_FILE.FITS`, `TEST_OBS.XML` and `COLLECTION`. Replace these values appropriately when executing the commands.
+
+1. Copy the file `TEST_FILE.FITS` in the working directory. The metadata in this file will be described in the CAOM Observation created during this example.
+
+1. The example will cause an instance to be created in the [CAOM2 sandbox](http://sc2.canfar.net/search/).  If you visit prior to the creation of the first CAOM instance for a `COLLECTION`, that `COLLECTION` will not show in the `Additional Constraints -> Collection` . Even after successful creation of a CAOM instance, it can take up to one day for the `COLLECTION` to be selectable from the UI.
+
+    To use production CADC services, remove `resource-id` parameters in `caom2-repo` commands.
+
+1. Install the following python dependencies:
+
+    ```
+    pip install caom2repo
+    pip install caom2utils
+    ```
+
+1. Get credentials organized. The examples assume the use of a [./.netrc file](https://www.systutorials.com/docs/linux/man/5-netrc/). The examples expect this file to be named `./.netrc`, located in the working directory, and with permissions set to `-rw-------`. The `./.netrc` file content should include the following, with cadcusername and cadcpassword replaced with your CADC username and password values:
+
+    ````
+    machine www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca login canfarusername password canfarpassword
+    machine www.canfar.net login canfarusername password canfarpassword
+    machine ws-cadc.canfar.net login canfarusername password canfarpassword
+    machine sc2.canfar.net login canfarusername password canfarpassword
+    ````
+    
+    To set the `./.netrc` file permissions:
+    
+    ```
+    chmod 600 ./.netrc
+    ```
+
+1. The caom2-repo client also supports username/password and X509 certificates. If you want to use X509 
+certificates use the --cert parameter instead of the -n parameter in all the commands. The command line client `cadc-get-cert` is installed with the prerequisites for the `caom2repo` package, and `cadc-get-cert --help` from a terminal prompt will describe how to obtain a CADC certificate.
+
+
+1. Test the install. Commands are case-sensitive.
+
+    ```
+    caom2-repo read --netrc ./.netrc --resource-id ivo://cadc.nrc.ca/sc2repo COLLECTION abc
+    ```
+
+    If the install was successful, this will report an error:
+
+    ```
+    Client Error: Not Found for url: http://sc2.canfar.net/sc2repo/auth-observations/COLLECTION/abc.
+    ```
+
+## Troubleshooting
+
+1. If `pip install caom2utils` fails with the following error:
+
+    ```
+    AttributeError: module ‘enum’ has no attribute ‘IntFlag’
+    ```
+
+    Ensure the version of vos is >= 3.1.1:
```

Comment:
I did update the setup.cfg file for caom2utils, but I'll leave the trouble-shooting in the docs, because of the pip uninstall that's required, and not necessarily obvious.